### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by MariiaMalitska**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,50 +374,41 @@ import { PrivateKeyStorage } from 'virgil-sdk';
...
-  * [Setup Card Manager][_card_manager] to manage user's Public Keys
-  * [Setup Card Verifier][_card_verifier] to verify signatures inside of user's Card
-  * [Setup Key storage][_key_storage] to store Private Keys
-  * [Setup your own Crypto library][_own_crypto] inside of the SDK
-* [More usage examples][_more_examples]
-  * [Create & publish a Card][_create_card] that has a Public Key on Virgil Cards Service
-  * [Search user's Card by user's identity][_search_card]
-  * [Get user's Card by its ID][_get_card]
-  * [Use Card for crypto operations][_use_card]
-* [API Reference][_reference_api]
```
Comment (xlwknx) [inline README.md]: please return it

File: README.md [hunk 2]
```diff
@@ -1,31 +1,39 @@
 > This README is for Virgil SDK v6. If you're here for the previous version (v5.3.x), check the [v5.3.x branch](https://github.com/VirgilSecurity/virgil-sdk-javascript/tree/v5.3.x) for Virgil SDK v5.3.x docs.
 
-# Virgil Security JavaScript SDK
+# Virgil Core SDK JavaScript 
 
-[![npm](https://img.shields.io/npm/v/virgil-sdk.svg)][npmjs]
+[![npm](https://img.shields.io/npm/v/virgil-sdk.svg)](https://www.npmjs.com/package/virgil-sdk)
 [![Build status](https://img.shields.io/travis/VirgilSecurity/virgil-sdk-javascript.svg)](https://img.shields.io/travis/VirgilSecurity/virgil-sdk-javascript.svg)
 [![GitHub license](https://img.shields.io/badge/license-BSD%203--Clause-blue.svg)](https://github.com/VirgilSecurity/virgil/blob/master/LICENSE)
```
Comment (xlewkanx) [inline README.md:9]: Add [API Reference](https://virgilsecurity.github.io/virgil-sdk-javascript/) please
```
[![API Reference](https://img.shields.io/badge/API%20Reference-virgil--sdk--javascript-green)](https://virgilsecurity.github.io/virgil-sdk-javascript//)
```

File: README.md (hunk 1 above)
Comment (xlewkanx) [inline README.md:223]: `[API Reference][_reference_api]` leave please

File: README.md [hunk 3]
```diff
@@ -1,31 +1,39 @@
...
-- use Virgil [Crypto library][_virgil_crypto]
-- use your own Crypto
+- Communicate with [Virgil Cards Service](https://developer.virgilsecurity.com/docs/platform/api-reference/cards-service/)
+- Manage users' public keys
+- Encrypt, sign, decrypt and verify data
+- Store private keys in secure local storage
+- Use Virgil [Crypto Library](https://github.com/VirgilSecurity/virgil-crypto-javascript)
+- Use your own crypto library
+
+## Crypto Library Purposes
```
Comment (xlewkanx) [inline README.md]: Are you sure we need this? Maybe just link to virgil-crypto?

File: README.md [hunk 4]
```diff
@@ -71,14 +79,181 @@ Note that the `virgil-crypto` script must also be added to the page.
...
+app.get('/virgil-jwt', (req, res) => {
+  // Get the identity of the user making the request (this assumes the request
+  // is authenticated and there is middleware in place that populates the
+  // `req.user` property with the user record).
+  // Identity can be anything as long as it's unique for each user (e.g. email,
+  // name, etc.). You can even obfuscate the identity of your users so that
+  // Virgil Security doesn't know your actual user identities.
+  const jwt = jwtGenerator.generateToken(req.user.email);
+  req.send(jwt.toString());
+});
```
Comment (xlewkanx) [inline README.md:146]:
```js
const express = require('express');
const { VirgilCrypto, VirgilAccessTokenSigner } = require('virgil-crypto');
const { JwtGenerator } = require('virgil-sdk');

async function getJwtGenerator() {
  await initCrypto();

  const virgilCrypto = new VirgilCrypto();
  // initialize JWT generator with your App ID and App Key ID you got in
  // Virgil Dashboard and the `appKey` object you've just imported.
  return new JwtGenerator({
    appId: config.virgil.appId,
    apiKeyId: config.virgil.appKeyId,
    // import your App Key that you got in Virgil Dashboard from string.
    apiKey: virgilCrypto.importPrivateKey(config.virgil.appKey),
    // initialize accessTokenSigner that signs users JWTs
    accessTokenSigner: new VirgilAccessTokenSigner(virgilCrypto),
    millisecondsToLive:  20 * 60 * 1000 // JWT lifetime - 20 minutes (default)
  });
}

app.get('/virgil-jwt', (req, res) => {
  const generator = await generatorPromise;
  const virgilJwtToken = generator.generateToken(req.user.identity);

  res.json({ virgilToken: virgilJwtToken.toString() });
});
```

**Review by xlewkanx — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -1,31 +1,34 @@
...
 
 ## SDK Features
-- communicate with [Virgil Cards Service][_cards_service]
-- manage users' Public Keys
-- store private keys in secure local storage
-- use Virgil [Crypto library][_virgil_crypto]
-- use your own Crypto
+- Communicate with [Virgil Cards Service](https://developer.virgilsecurity.com/docs/platform/api-reference/cards-service/)
+- Manage users' public keys
+- Encrypt, sign, decrypt and verify data
```
Comment (xlewkanx) [inline README.md]: Crypto or e3kit feature, not sdk's

File: README.md [hunk 6]
```diff
@@ -1,31 +1,34 @@
...
+- Store private keys in secure local storage
+- Use Virgil [Crypto Library](https://github.com/VirgilSecurity/virgil-crypto-javascript)
+- Use your own crypto library
+- **Post quantum algorithms support**. [Round5](https://round5.org/) (ecnryption) and [Falcon](https://falcon-sign.info/) (signature)
```
Comment (xlewkanx) [inline README.md]: Also crypto feature, not sdk's

Comment (xlewkanx) [reply README.md]: You can add something about JWT Generation

File: README.md [hunk 7]
```diff
@@ -1,31 +1,34 @@
...
 ## SDK Features
-- communicate with [Virgil Cards Service][_cards_service]
-- manage users' Public Keys
-- store private keys in secure local storage
-- use Virgil [Crypto library][_virgil_crypto]
-- use your own Crypto
+- Communicate with [Virgil Cards Service](https://developer.virgilsecurity.com/docs/platform/api-reference/cards-service/)
+- Manage users' public keys
+- Encrypt, sign, decrypt and verify data
+- Store private keys in secure local storage
```
Comment (xlewkanx) [inline README.md]: JS storages are not secure. You can write that we use `IndexedDB`as browser storage and filesystem for node.js. Also we have https://github.com/VirgilSecurity/virgil-key-storage-rn

File: README.md [hunk 8]
```diff
@@ -71,14 +74,181 @@ Note that the `virgil-crypto` script must also be added to the page.
...
+const cardManager = new CardManager({
+  cardCrypto: cardCrypto,
+  accessTokenProvider: accessTokenProvider,
+  cardVerifier: cardVerifier
+});
+```
+
+### Set up Key Storage for private keys
+
+This subsection shows how to set up a `VSSKeyStorage` using Virgil SDK in order to save private keys after their generation.
```
Comment (xlewkanx) [inline README.md]: Where did you get this: `VSSKeyStorage` ?

Comment (xlewkanx) [reply README.md]: I think this is about `PrivateKeyStorage `

**Review by xlewkanx — APPROVED** [review]

**Pull request merged**